test/parallel/test-stringbytes-external-at-max.js

```diff
@@ -1,22 +1,27 @@
 'use strict';
+// Flags: --expose-gc
 
 require('../common');
 const assert = require('assert');
 
+assert(typeof gc === 'function', 'Run this test with --expose-gc');
+
 // v8 fails silently if string length > v8::String::kMaxLength
 // v8::String::kMaxLength defined in v8.h
 const kStringMaxLength = process.binding('buffer').kStringMaxLength;
 
+var buf;
 try {
-  new Buffer(kStringMaxLength * 3);
+  buf = new Buffer(kStringMaxLength);
+  // Try to allocate memory first then force gc so future allocations succeed.
+  new Buffer(2 * kStringMaxLength);
```

Comment:
You're probably aware of this but you don't need to declare `var buf` upfront, you can just do it here.  Mostly a matter of taste but doing `var buf = ...` is arguably a little more idiomatic w.r.t. the "house style."